### ISSUE TITLE
Implement javascript code for the context tabledrag

### ISSUE THREAD

**Issue by NormySan** [oddhill/d8-context#3]
The context entity list tabledrag needs javascript that will update the group and weights like on the blocks form.

**Comment by njt1982** [comment]
![context___indigoblue](https://cloud.githubusercontent.com/assets/384976/11722221/8ae8cf86-9f5f-11e5-9fd4-bfd4750eef16.png)

Is that this bug?

**Comment by NormySan** [comment]
@njt1982 No, I've not had the time to implement the Javascript for this.
